ast/switch_statement.go

```diff
@@ -0,0 +1,30 @@
+package ast
+
+import "bytes"
+
+type SwitchStatement struct {
+	*Meta
+	Control Expression
+	Cases   []*CaseStatement
+	Default int
```

Comment:
`Default` seems that `switch` statement has `default` or not and indicates index of `Cases` field as default statement but I would suggest it should be `*CaseStatement` and set on parse `DEFAULT` token particularly.

Reply:
I implemented it this way with the index due to the fact that the default case can be fallen into and out of.

This is valid and will have log output from all 3 cases:
```
set req.http.foo = "1";
switch (req.http.foo) {
  case "1":
    log "case 1";
    fallthrough;
  default:
    log "default";
    fallthrough;
  case "2":
    log "case 2";
    break;
}
```

The default case could be stored outside of the case list, but it would complicate implementation of this fallthrough behavior.

Reply:
Understood, to think of `fallthrough` behavior, your way would be better.